src/injectIntoComponent.js

```diff
@@ -0,0 +1,46 @@
+import { PropTypes } from 'react';
+import Injector from './Injector';
+/**
+ * Creates an extended component that will use injector-js DI to inject into the context.
+ * @param  {React.Component} Component The Component To Extend
+ * @return {React.Component}           Extended Component
+ */
+export default function injectIntoComponent(Component) {
```

Comment:
@bassettsj Should be just the same, the `ComposedComponent` in this example is the `Component` in your example.